docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -945,7 +945,7 @@ See an example in [this PR that upgrades the `using-remark` site to Gatsby v2](h
 
 If your Gatsby v1 site included any polyfills, you can remove them. Gatsby v2 ships with Babel 7 and is configured to automatically include polyfills for your code. See [Gatsby's Babel docs for more details](/docs/babel).
 
-> Note: This works for your own code, but is not yet implemented for code imported from `node_modules`. Track progress of this feature in [this issue](https://github.com/gatsbyjs/gatsby/issues/7064).
+> Note: This works for your code, but is not yet implemented for code imported from `node_modules`. Track the progress of this feature in [this issue](https://github.com/gatsbyjs/gatsby/issues/7064).
```

Comment:
You can remove this note as it has been implemented

Reply:
So, should I remove the present note ?

Reply:
Yes! :)